zephyr/Kconfig

```diff
@@ -18,6 +18,30 @@ config LV_Z_USE_FILESYSTEM
 	help
 	  Enable LittlevGL file system
 
+config LV_Z_FLUSH_THREAD
+	bool "Flush LVGL frames in a separate thread"
```

Comment:
Any reason not to enable it by default?

Reply:
See here: https://github.com/zephyrproject-rtos/zephyr/pull/59166#issuecomment-1590193173